Documentation/artifact-format.md

```diff
@@ -281,29 +281,30 @@ Format: JSON
 
 ```
 {
-  "updates": [
-    {
-      "type": "rootfs-image"
-    },
-    {
-      "type": "rootfs-image"
+    "updates": [
+        {
+            "type": "rootfs-image"
+        },
+        {
+            "type": "delta-image"
+        }
+    ],
+    "ArtifactProvides": {
```

Comment:
Shouldn't artifact-provides be an object? Or should this be a list of artifact-provides?